tests/components/guardian/test_diagnostics.py

```diff
@@ -4,15 +4,16 @@
 from homeassistant.components.guardian import DOMAIN, GuardianData
 from homeassistant.core import HomeAssistant
 
+from tests.common import MockConfigEntry
 from tests.components.diagnostics import get_diagnostics_for_config_entry
 from tests.typing import ClientSessionGenerator
 
 
 async def test_entry_diagnostics(
     hass: HomeAssistant,
-    config_entry,
+    config_entry: MockConfigEntry,
     hass_client: ClientSessionGenerator,
-    setup_guardian,
+    setup_guardian: None,
```

Comment:
Should this be pytest.mark.usefixtures?

Reply:
I'm not sure why - but it doesn't work here.
I guess in this test the order of the fixtures is important.

Reply:
Ok

Reply:
Maybe add a comment?